src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -120,6 +121,7 @@
     private static final Attribute<GcvUsage> GCV_USAGE_ATTRIBUTE =
             Attribute.of("com.palantir.consistent-versions.usage", GcvUsage.class);
     private static final String GCV_LOCKS_CAPABILITY = "gcv:locks:0";
+    private static final String WRITE_VERSIONS_LOCK_TASK = "writeVersionsLock";
```

Comment:
Commence bikeshedding here.

Here are a few options and my thoughts:

1. `writeVersionsLock` - communicates clearly we're updating the `versions.lock` file, but is a bit of a mouthfull
2. `versionsLock` - a bit shorter than 1) but doesn't imply we're updating it
3. `gcvLocks` - if you know what GCV is and how it works, this is obvious, but maybe not as obvious as 1)
4. `writeDependencyLocks` - probably the most similar to the current `--write-locks`, although the use of "dependency" may not be as clear.

Reply:
I like the idea of a `writeLocks` prefix or suffix to group write-locks tasks together. Neither `writeLocksVersions` nor `versionsWriteLocks` exactly roll off the tongue though.

Reply:
So I suppose my preference is the sandwich pattern `write[identifier]Lock(s)?` Not sure if we want lock to be singular or plural, "write version locks" vs "write versions lock".

I think plural form "Locks" may be slightly better because some lock tasks update multiple files, e.g. `[module]/baseline-class-uniqueness.lock`, and we probably want the names to be as similar as possible to reduce mental overhead.

I don't have a terribly strong opinion here, just typing out my thoughts.

Reply:
+1 to writeVersionsLocks

Reply:
I've gone with `writeVersionsLocks` as with gradle-like task abbreviation all of these work:

```
writeVersionsLocks
writeVersionsLock
writeVersionLocks
wVL
```